packages/web/src/stores/tracks.ts

```diff
@@ -126,19 +131,24 @@ export const tracksStore = defineStore('tracks', () => {
     // console.log('store.tracks > addToUsedTracks > srcTrackId : ', srcTrackId)
     // console.log('store.tracks > addToUsedTracks > newTrackId : ', newTrackId)
 
-    const track = getTrack(srcTrackId)
+    // const srcTrack = getTrack(srcTrackId)
+    // console.log('store.tracks > addToUsedTracks > srcTrack : ', srcTrack)
+    const nextTrack = getTrack(newTrackId)
+    // console.log('store.tracks > addToUsedTracks > nextTrack : ', nextTrack)
 
     removeFurtherUsedTracks(srcTrackId)
 
     // add newTrackId
     const trackInfos: UsedTrack = {
       id: newTrackId,
-      category: track?.category,
+      component: nextTrack?.interface.component || TrackComponents.buttons,
```

Comment:
question: @yvalentin I have a TS error here, prohibiting the build. The `component` key is indeed declared in `UsedTrack` type as `enum`

The error tooltip I got in my editor is this => 

![Capture d’écran 2023-10-26 à 11 19 04](https://github.com/betagouv/transition-ecologique-entreprises-widget/assets/21986727/ea25b708-c462-4385-addd-654e792e7bba)

Any clue ?